.ci/install_crio.sh

```diff
@@ -93,10 +93,8 @@ if [ "$ID" == "centos" ] || [ "$ID" == "fedora" ]; then
 	# This is necessary to avoid crashing `make` with `No package devmapper found`
```

Comment:
Hi @fidencio, I am wondering if this block can also be removed.

Reply:
By this block, you mean disable the `selinux_tag` check? it was added a long time ago and, indeed, I think we can just drop it entirely.

Reply:
yes, drop the selinux_tag check entirely.